build.gradle

```diff
@@ -43,8 +43,8 @@ ext {
     jsonVersion = '20160810'
     junitVersion = '4.10'
     lombokVersion = '1.18.10'
-    mantisClientVersion = '1.3.+'
-    mantisDiscoveryProtoVersion = '1.3.+'
+    mantisClientVersion = '1.3.52'
```

Comment:
why pin to 52?

Reply:
I think the BOM recommendation is using a much newer version and nfmaintis-api does not compile. I think a better way is to create dependency lock for this repo, and we manage dependency versions via the lock file as other Mantis repositories. The `master` branch recently introduced the dependency lock file, but that is using 2.0 mantis-client and not deployable.